pkg/plugin/plugin.go

```diff
@@ -0,0 +1,286 @@
+package plugin
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+type LocalVolumeObjectStore struct {
+	log        logrus.FieldLogger
+	volumeType VolumeType
+}
+
+// NewFileObjectStore instantiates a LocalVolumeObjectStore.
+func NewLocalVolumeObjectStore(log logrus.FieldLogger, v VolumeType) *LocalVolumeObjectStore {
+	return &LocalVolumeObjectStore{
+		log:        log,
+		volumeType: v,
+	}
+}
+
+// Init initializes the plugin. It can be called multiple times.
+func (o *LocalVolumeObjectStore) Init(config map[string]string) error {
+	bucket := config["bucket"]
+	path := filepath.Join(getRoot(), bucket)
+
+	log := o.log.WithFields(logrus.Fields{
+		"bucket": bucket,
+		"path":   path,
+	})
+	log.Debug("LocalPvObjectStore.Init called")
```

Comment:
is this debug still needed?

Reply:
Debug messages will only appear if the debug flag is passed into the velero server. Could be useful for debugging calls to the plugin. Do you think this is still too much info?